### DISCUSSION TITLE
Midi Breakout Schematic

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#298] [General]
Hi!

I see on the Wiki the Midi RX / TX circuit for flashed boards, which seems to be the same one used for any type of compatible board.

Does anyone know if these commercial ones have the same circuit, and could be dropped in and used rather than building the circuit from the schematic?

https://www.aliexpress.us/item/3256806831934101.html?spm=a2g0o.productlist.main.15.31d753a2qExagv&algo_pvid=29d65184-5308-41f7-8768-640c2dbeb809&algo_exp_id=29d65184-5308-41f7-8768-640c2dbeb809-7&pdp_npi=4%40dis%21USD%2116.13%214.52%21%21%21116.85%2132.72%21%402103146c17188905804416233e5e58%2112000039095717199%21sea%21US%213739075724%21&curPageLogUid=lvOy2UrdDOps&utparam-url=scene%3Asearch%7Cquery_from%3A

Thanks!

**Comment by JimMcDougall** [comment]
That board uses a 6N138 opto isolator that is designed for 5V use. It will not do well with 3.3v

**Reply by NRKorsakov** [reply]
Hi thanks for that! I only need output, os I'll just do the connector/resistor thing.
